### PR TITLE
try to improve text inflation

### PR DESCRIPTION
**Pull request by AndyOGo**
improves #237 according to https://developer.mozilla.org/en-US/docs/Web/CSS/text-size-adjust#Browser_compatibility

### CODE REVIEW

File: src/styles/_reboot.scss [hunk 1]
```diff
@@ -38,8 +38,14 @@
 html {
   font-family: sans-serif; // 2
   line-height: 1.15; // 3
+  -webkit-text-size-adjust: none; // 4
```
Comment (LucaMele) [inline src/styles/_reboot.scss]: vendor prefix shoukld be done by the post css which we have. please try without and ispoect the css

Comment (LucaMele) [inline src/styles/_reboot.scss]: vendor prefix shoukld be done by the post css which we have. please try without and ispoect the css

Comment (AndyOGo) [reply src/styles/_reboot.scss]: right, I will take a deep look at postCSS support for `text-size-adjust`

Comment (AndyOGo) [reply src/styles/_reboot.scss]: Okay, `text-size-adjust` support seems to be there https://github.com/postcss/autoprefixer/issues/264

Expect Firefox for Android, which isn't include in the default browser list
https://github.com/postcss/autoprefixer/issues/496

**Pull request merged**